label_studio/io_storages/all_api.py

```diff
@@ -146,3 +151,8 @@ def list(self, request, *args, **kwargs):
             [self._get_response(s['export_list_api'], request, *args, **kwargs) for s in _common_storage_list], []
         )
         return Response(list_responses)
+
+    def initial(self, request, *args, **kwargs):
```

Comment:
I think we also need to add the similar check in PATCH request in ImportStorageDetailAPI and ExportStorageDetailAPI